manual/src/threelang/tfgrid/funkwhale.md

```diff
@@ -0,0 +1,33 @@
+# Funkwhale Namespace
+
+- To deploy a funkwhale instance, use the funkwhale namespace.
+
+## Create Operation
+
+- action name: !!tfgrid.funkwhale.create
+- parameters:
+  - model_name [required]
+  - farm_id [optional]
+    - if 0, machines could span multiple nodes on different farms
+  - capacity [required]
+    - a string in ['small', 'medium', 'large'] indicating the capacity of the machines
+    - small: 2 vCPU, 1GB RAM, 50GB SSD
+    - medium: 2 vCPU, 4GB RAM, 100GB SSD
+    - large: 4 vCPU, 8GB RAM, 250GB SSD
```

Comment:
i was thinking about unifying capacity settings to not confuse users, it would make it easier right?
```
- a string in ['small', 'medium', 'large', 'extra-large'] indicating the capacity of the discourse instance
  - small: 1 vCPU, 2GB RAM, 10GB SSD
  - medium: 2 vCPU, 4GB RAM, 50GB SSD
  - large: 4 vCPU, 8GB RAM, 240 SSD
  - extra-large: 8vCPU, 16GB RAM, 480GB SSD
```
maybe we could modify these settings if they are not that good, but i would suggest unifying them for all solutions